03/app.test.js

```diff
@@ -1 +1,42 @@
-import randomNumber from './app';
\ No newline at end of file
+import randomNumber from "./app";
+
+describe("Tests for min and max", () => {
+  it("return 1 when draw numbers from 1 to 1", () => {
+    expect(randomNumber(1, 1)).toBe(1);
+  });
+
+  it("throw exception when max is smaller than min", () => {
+    function useRandomNumber() {
+      randomNumber(2, 1);
+    }
+    expect(useRandomNumber).toThrow();
+  });
+});
+
+describe("Tests for min", () => {
+  it("throw exception when min is not a number", () => {
+    function useRandomNumber() {
+      randomNumber("string", 1);
+    }
+    expect(useRandomNumber).toThrow();
+  });
+
+  it("check if drawn numer is equal or grater then min", () => {
+    const drawnNumber = randomNumber(1, 6);
+    expect(drawnNumber).toBeGreaterThanOrEqual(1);

```

Comment:
Można w jednym miejscu sprawdzić zakres tj.
```
expect(drawnNumber).toBeGreaterThanOrEqual(1);
 expect(drawnNumber).toBeLessThanOrEqual(6);
```
PS. To co napisałaś jest ok - tylko informacyjnie piszę :)